day_7/10_speckled_frogs.js

```diff
@@ -0,0 +1,45 @@
+//Main Task - 10 Speckled Frogs
+var num1 = [10, 9, 8, 7, 6, 5, 4, 3, 2, 1];
+var num2 = [9, 8, 7, 6, 5, 4, 3, 2, 1, "no more"]
```

Comment:
When you have very similar values like this, it's a sign of repeated code that could be abstracted. Also, storing a string in an array of numbers is an anti-pattern. You can use basic arithmetic in the brackets like `num[i+1]` if you want to get at an index value one place over.